rockwire.yaml

```
@@ -0,0 +1,425 @@
+openapi: 3.0.0
+info:
+  title: Rockwire Platform APIs
+  description: This is REST API design and documentation for Rockwire platform
+  version: 1.0.0
+servers:
+- url: https://virtserver.swaggerhub.com/uiuc-scci/profiles/1.0.0
+  description: SwaggerHub API Auto Mocking
+tags:
+- name: profiles
```

Let's use singular for this section heading:
```suggestion
- name: Profile
```